core/src/main/java/com/bankbazaar/tripmanager/manager/TripActivityManager.java

```diff
@@ -55,25 +56,23 @@ private Optional<TripActivity> exists(Long id)
         return tripActivityRepository.findById(id);
     }
 
-    public TripActivity updateData(TripActivity presentData,TripActivity tripActivity)
+    public void updateData(TripActivity presentData,TripActivity tripActivity)
     {
         if(tripActivity.getActivityStatus()!=null)
         {
             presentData.setActivityStatus(tripActivity.getActivityStatus());
         }
-        if(tripActivity.getActivityDescription()!=null)
+        if(!tripActivity.getActivityDescription().isBlank())
```

Comment:
This will throw NullPointerException if getActivityDescription returns null

Reply:
all class members are set to nullabale=false. wouldn't that remove any case of it being null?